### PR TITLE
Clean up CSS; replace div.circle and div.gap-patch with ::after

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

**Comment by keanulee** [comment]
Experimenting to see if I can replace .gap-patch with .spinner-layer::after as well

File: paper-spinner.html [hunk 1]
```diff
@@ -53,48 +53,32 @@
...
-        <div class="circle-clipper left">
-          <div class="circle"></div>
-        </div><div class="gap-patch">
-          <div class="circle"></div>
-        </div><div class="circle-clipper right">
-          <div class="circle"></div>
-        </div>
+        <div class="circle-clipper left"></div>
+        <div class="gap-patch"></div>
+        <div class="circle-clipper right"></div>
```
Comment (frankiefu) [inline paper-spinner.html]: 👍

**Comment by frankiefu** [comment]
Perf test shows `paper-spinner` is about 18% faster to create/render and `paper-spinner-lite` is about 10% faster.  That's awesome!

**Comment by abdonrd** [comment]
Awesome! 🎉

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**